demo-repository/exercises/demo/descr.html

```diff
@@ -3,7 +3,7 @@
 </p>
 
 <h2>The task</h2>
-
+<img src="/static/demo/files/OCaml.png" alt="OCaml">
```

Comment:
@CJs0800 I tested deploying the branch, and it appears the image is way too large (the width is not automatically limited).

So I propose this:
```suggestion
<img src="/static/demo/files/OCaml.png" alt="OCaml" width="400px"/>
```